### DISCUSSION TITLE
Splitting Simple Compound Sentences

### DISCUSSION THREAD

**Discussion by YaCnDehfuli** [explosion/spaCy#11122] [Help: Coding & Implementations] [answered]
Hello All!

I am new to spacy so sorry about the possible mistakes.  I just wanted to split some commands like ("Turn on the lights and play some music") to ("Turn on the lights" and " Play some music") in order to give them to RASA for intent classifications and entity recognition (p.s.: I know about rasa multiple intent recognition but I can not use it).

The Spacy .sent function considers this as one sentence. What should  I do?

**Comment by polm** [comment, marked as answer]
What you're trying to do is called **sentence simplification** and unfortunately it's not actually that simple in the general case - it's an ongoing research topic. 

A simple thing you can do is look for words with the `cc` dependency and split the sentence there, as outlined in #3407. Using similar rules for specific situations, you should be able to handle sentences like your example and get pretty good coverage. 

For the more general case, or more difficult cases, check [NLP Progress on Sentence Simplification](https://github.com/sebastianruder/NLP-progress/blob/master/english/simplification.md).
